pkgs/development/libraries/wlroots/default.nix

```diff
@@ -23,9 +23,13 @@ stdenv.mkDerivation rec {
   buildInputs = [
     libGL wayland wayland-protocols libinput libxkbcommon pixman
     xcbutilwm libX11 libcap xcbutilimage xcbutilerrors mesa
-    libpng ffmpeg
+    libpng ffmpeg libuuid xorg.xcbutilrenderutil xwayland
   ];
 
+  prePatch = ''
+    sed -i 's/0.12.0/0.13.0/' meson.build
```

Comment:
Use `substituteInPlace` here.

(To be honest I also prefer sed, but using the nixpkgs-provided functions is the preferrable way.)

Reply:
This is going to be squashed away when wlroots 0.13 is released